### ISSUE TITLE
V5 tests : inferences : have more recursivity inside the same type of objects

### ISSUE THREAD

**Issue by manonviallet** [LuatixHQ/opencti#1628]
## Description

When testing V5 some inferences scenario do not go back far enough, especially for some types were several objects of the same type can be "nested" (russian dolls style) : this is the case for location (city -> located in country -> located in sub-region -> located in region OR threats actors belonging to threats actors).
In those cases, inferences should take into account the full chain : for instance, an incident targeting a city : the inference should go back as far as the region. Idem for a threat actor to which is attributed an intrusion set, it should be inferred that the intrusion set is attributed to the full chain of threat actors belonging to each others.

## Environment

2. OpenCTI V5.0.1
3. OpenCTI frontend

**Comment by SamuelHassine** [comment]
Hello @manonviallet,

We tried several times to reproduce the problem without success. Can you provide us with a clear step-by-step reproduction case?

Thanks!

**Comment by manonviallet** [comment]
Sorry, so I investigated again to have more clarity (we also discovered issues in our database so it made everything muddy) and here are the following. My issue is the wrong one as the problem is not what I first described. There are actually two different and distinct things, which are features issues. I opened dedicated issues and will close this one.
See https://github.com/OpenCTI-Platform/opencti/issues/1662
And https://github.com/OpenCTI-Platform/opencti/issues/1663
For location, it works really well.
Sorry for the inconvenience.
